superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -34,6 +34,7 @@ import CrossFilterScopingModal from 'src/dashboard/components/CrossFilterScoping
 import Icons from 'src/components/Icons';
 import ModalTrigger from 'src/components/ModalTrigger';
 import ViewQueryModal from 'src/explore/components/controls/ViewQueryModal';
+import { FileImageOutlined, FileOutlined } from '@ant-design/icons';
```

Comment:
Shouldn't these come from our `Icons` component?

Reply:
Yeah, it got auto-imported when I copy pasted from another file. Fixed